src/compas_timber/connections/joint.py

```diff
@@ -259,3 +259,126 @@ def _beam_side_incidence(beam_a, beam_b, ignore_ends=True):
             face_angles[face_index] = angle_vectors(face.normal, centerline_vec)
 
         return face_angles
+
+    @staticmethod
+    def _beam_ref_side_incidence(beam_a, beam_b, ignore_ends=True):
```

Comment:
not for this PR but in the future we should consider taking these static methods and move them to a new module (something like `joinery_toolbox`) as module functions.